examples/bri-3/src/bri/identity/bpiSubjectAccounts/capabilities/updateBpiSubjectAccount/updateBpiSubjectAccountCommand.handler.ts

```diff
@@ -20,6 +25,12 @@ export class UpdateBpiSubjectAccountCommandHandler
 
     this.agent.updateBpiSubjectAccount();
 
-    await this.storageAgent.updateBpiSubjectAccount(bpiSubjectAccountToUpdate);
+    return this.mapper.map(
```

Comment:
lets split into two explicit lines for clarity
var x = await this.storageAgent.update..
return this.mapper.map(x,...)